tests/Doctrine/Tests/DBAL/Functional/Schema/SqliteSchemaManagerTest.php

```diff
@@ -232,4 +232,63 @@ public function getDiffListIntegerAutoincrementTableColumnsData()
             array('bigint', true, true),
         );
     }
+
+    /**
+     * @group DBAL-2921
+     */
+    public function testPrimaryKeyAutoIncrement()
+    {
+        $table = new Schema\Table('test_pk_auto_increment');
+        $table->addColumn('id', 'integer', [
+            'autoincrement' => true,
+        ]);
+        $table->addColumn('text', 'text');
+        $table->setPrimaryKey(['id']);
+        $this->_sm->dropAndCreateTable($table);
+
+        $this->_conn->insert('test_pk_auto_increment', ['text' => '1']);
+        $this->_conn->insert('test_pk_auto_increment', ['text' => '2']);
+        $this->_conn->insert('test_pk_auto_increment', ['text' => '3']);
```

Comment:
If one record is enough to test that autoincrement works in fact, please leave one out of three.

Reply:
Good point. I removed two lines in this test and in `testPrimaryKeyNoAutoIncrement`.